pkgs/servers/isso/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, buildPythonApplication
+, misaka, werkzeug, ipaddr, configparser, html5lib_0_9999999 }:
+
+let
+  misaka_1 = misaka.overridePythonAttrs (old: rec {
```

Comment:
Better specify `python` as an argument and override as in https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/home-assistant/default.nix.
This will make sure that all dependencies use the old version of misaka as well (which is of course not necessary right now but might be in the future).